core-web/libs/dotcms-scss/angular/_variables.scss

```diff
@@ -45,7 +45,7 @@ $info: $color-palette-secondary;
 
 // Basic stuffs
 $field-tiny-height: 28px;
-$field-disabled-opacity: 0.7;
+$field-disabled-opacity: 0.38;
```

Comment:
this variable was out of use, didn't find any reference in the project, so I updated and use it.